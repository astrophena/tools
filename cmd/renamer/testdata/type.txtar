-- a.md --
foo
-- b.md --
bar
-- c.txt --
foobar
-- d.txt --
barfoo
