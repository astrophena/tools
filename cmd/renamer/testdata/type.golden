-- 1.md --
foo
-- 2.md --
bar
-- 3.txt --
foobar
-- 4.txt --
barfoo
